dist/Caddyfile

```diff
@@ -0,0 +1,32 @@
+example.com {
+	root * /home/mastodon/live/public
+
+	@staticfiles file
+	handle @staticfiles {
+		file_server
+	}
+
+	reverse_proxy /api/v1/streaming* :4000
+
+	reverse_proxy :3000
```

Comment:
The only gripe I would have with this is that I'd make the host explicit here instead of relying on the implicit idea of "this lives on the same server as I do". Otherwise it's a great `Caddyfile`.